.github/workflows/linux-build.yml

```
@@ -50,9 +49,122 @@ permissions:
+      CCACHE_DIR: "${{ github.workspace }}/ccache"
+      USE_CLANG: "${{ 'false' }}"
+    defaults:
+      run:
+        shell: bash
+        working-directory: velox
+    steps:
+      - name: Get Ccache Stash
+        uses: assignUser/stash/restore@v1
+        with:
+          path: '${{ env.CCACHE_DIR }}'
+          key: ccache-ubuntu-debug-default-${{ inputs.use-clang && 'clang' || 'gcc' }}
```

actionlint (1.7.7)

property "use-clang" is not defined in object type {}

(expression)